assets/src/blocks/amp-story/edit.js

```diff
@@ -186,6 +210,25 @@ class EditPage extends Component {
 							) }
 						</Fragment>
 					</PanelBody>
+					<PanelBody title={ __( 'Page Settings', 'amp' ) }>
+						<SelectControl
+							label={ __( 'Advance to next page', 'amp' ) }
+							help={ 'media' === autoAdvanceAfter ? __( 'Based on the first media block encountered on the page', 'amp' ) : undefined }
+							value={ autoAdvanceAfter }
+							options={ autoAdvanceAfterOptions }
+							onChange={ ( value ) => setAttributes( { autoAdvanceAfter: value } ) }
```

Comment:
Should changing from 'time' to 'auto' trigger adjusting the `autoAdvanceAfterDuration` value? If I understand correctly then "Auto" would use automatically generated value based on animations, looks like `time` and `auto` are sharing `autoAdvanceAfterDuration` for `save`, currently it looks like if switching from `auto` the previous `time` value is still used.

Reply:
> currently it looks like if switching from auto the previous time value is still used.

Indeed. It should be trivial to fix, as we already have the total animation time available here. Will add this.